src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/ConfigurationBinderTests.Generator.cs

```diff
@@ -8,8 +8,14 @@ namespace Microsoft.Extensions.SourceGeneration.Configuration.Binder.Tests
 {
     public partial class ConfigurationBinderTests : ConfigurationBinderTestsBase
     {
+        /// <summary>
+        /// This is a regression test for https://github.com/dotnet/runtime/issues/90851.
+        /// It asserts that the configuration binding source generator properly formats
+        /// binding invocation source locations that the generated interceptors replace.
+        /// Do not modify these tests if the change reduces coverage for this scenario.
+        /// </summary>
         [Fact]
-        public void GeneratorHandlesInvocationsOnNewline()
+        public void TestBindingInvocationsOnNewLines()
         {
             IConfiguration configuration = TestHelpers.GetConfigurationFromJsonString(@"{""Longitude"":1,""Latitude"":2}");
 
```

Comment:
I suggest add inline comment for different cases. doing that ensure anyone touching the code will read the comment :-)

Reply:
I don't think any individual test case has interesting details specific to its syntax. We are just exercising various legal C# invocation syntax, in the spirit of fuzz testing (which btw would be overkill IMO).

IOW any inline comments would have the same general info for every test case.

Reply:
I agree with @tarekgh's suggestion. These tests need to have comments such as, "This tests the scenario of a newline between the OptionsBuilder instance reference and `.Bind`," or, "Place a newline between the `.` and `BindConfiguration`."

As is, the comment above just says not to modify the tests, but it doesn't cite what key elements cannot be modified. Presumably, modifications to the variable names would be OK while it's the very specific newline placements that need to be retained for coverage. If the comments feel repetitive, then the comments likely have room for being more specific to the condition being tested.

Reply:
Thanks folks; added some more comments.